### ISSUE TITLE
Lecture 7 Branching.Rmd 

### ISSUE THREAD

**Issue by pbondarenko2** [uofi-stat430/fa20-disc#13]
Hi Prof. Eddelbuettel, 

In Video 7 (Git branching), you copy and pasted code from a Markdown file called Branching.Rmd. Is this file posted on the course website? 

Thanks,
Polina

**Comment by eddelbuettel** [comment]
Good catch, thank you for the heads-up!  I copied the file over for the slide update, but never linked to it from the site.  

Now added at https://stat430.com/lectures/week02/ --- and box.com is a little funny with ending in .Rmd.  The first link will download the file, and your computer will then open the corresponding app for an Rmd file (which is RStudio on my computers).  The second link is renamed as .Rmd.txt and box.com will show you the raw content in the browser.  What is in there is the topic of lecture 5 for this week...

**Comment by eddelbuettel** [comment]
Direct links to file (which still require U of I sign-on to uofi.box.com):
- [Rmd](https://uofi.box.com/s/xi6r2q05qf75g5lpo5wnjv0422liptfu) 
- [Rmd.txt](https://uofi.box.com/s/y2497sukxf90u7u0cwkw3nn6f0znzd6z)
